lib/sw-precache.js

```diff
@@ -181,7 +181,7 @@ function generate(params, callback) {
     var runtimeCaching;
     var swToolboxCode;
     if (params.runtimeCaching) {
-      var pathToSWToolbox = require.resolve('sw-toolbox/sw-toolbox.js');
+      var pathToSWToolbox = require.resolve('sw-toolbox/build/sw-toolbox.js');
```

Comment:
Honest oversight :) IMO we should revert this change upstream (or revert, retag, intro as a major change). It's starting to cause issues to be filed on projects elsewhere (Polymer CLI etc)